src/styles/utils/_functions.scss

```
@@ -0,0 +1,23 @@
+@function minuteHandLeft($size) {
+  @return secondHandLeft($size) - 1px;
+}
+
+@function minuteHandWidth($size) {
+  @return $size / 60 + 2px;
+}
+
+@function minuteHandHeight($size) {
+  @return $size / 3;
+}
```

```suggestion
@function minuteHandLeft($size) {
  @return secondHandLeft($size) - 1px;
}

@function minuteHandWidth($size) {
  @return $size / 60 + 2px;
}

@function minuteHandHeight($size) {
  @return $size / 3;
}
```
Не понятно, зачем в этом задании такие сложности с размерами. Почему просто не задать размер родительскому блоку и затем давать процентные размеры внутренним элементам?